psutil/psutil_integration_test.go

```diff
@@ -30,13 +30,31 @@ import (
 	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
 )
 
+func TestPsutilGetMetricTypes(t *testing.T) {
+	Convey("psutil collector", t, func() {
+		p := &Psutil{}
```

Comment:
I think it's safer to use `NewPsutilCollector()` for creating plugin instance in tests, so that if some future versions of plugin require some additional setup for plugin to work, only change in `NewPsutilCollector()` will be needed.